src/platform/packages/shared/kbn-es-archiver/src/actions/empty_kibana_index.ts

```diff
@@ -17,7 +17,13 @@ export async function emptyKibanaIndexAction({ client, log }: { client: Client;
   const stats = createStats('emptyKibanaIndex', log);
 
   await cleanSavedObjectIndices({ client, stats, log });
-  await client.indices.refresh({ index: ALL_SAVED_OBJECT_INDICES });
+  // Refresh indices to prevent a race condition between a write and subsequent read operation. To
+  // fix it deterministically we have to refresh saved object indices and wait until it's done.
+  await client.indices.refresh({ index: ALL_SAVED_OBJECT_INDICES, ignore_unavailable: true });
+
+  // Additionally, we need to clear the cache to ensure that the next read operation will
+  // not return stale data.
+  await client.indices.clearCache({ index: ALL_SAVED_OBJECT_INDICES, ignore_unavailable: true });
```

Comment:
I need to do some more tests but locally it looks like commenting this out might fix the last broken FTR, not sure if this is required to fix other FTRs though. 

I will do some more testing after resolving a different unrelated issue that just came up.

Reply:
nm, it was not this.